java-util/src/main/java/org/apache/druid/java/util/metrics/HttpPostEmitterMonitor.java

```diff
@@ -81,6 +96,24 @@ private void emitTimeCounterMetrics(ServiceEmitter emitter, ConcurrentTimeCounte
     emitter.emit(builder.build(metricNameBase + "minTimeMs", timeCounter.getAndResetMinTime()));
   }
 
+  private void emitAllocatedBuffers(ServiceEmitter emitter)
+  {
+    int newTotalAllocatedBuffers = httpPostEmitter.getTotalAllocatedBuffers();
+    int allocatedBuffersDelta = newTotalAllocatedBuffers - lastTotalAllocatedBuffers;
+    emitter.emit(builder.build("emitter/buffers/allocated/total", newTotalAllocatedBuffers));
```

Comment:
This isn't compatible with the old name `emitter/buffers/totalAllocated`, but I think it's ok because we never documented the old names anyway.